tox.ini

```diff
@@ -87,6 +87,10 @@ envlist =
     py3{4,5,6,7,8}-test-ext-sqlite3
     pypy3-test-ext-sqlite3
 
+    ; opentelemetry-instrumentation-utils
+    py3{5,6,7,8}-test-ext-utils
```

Comment:
Should we rename to `test-instrumentation-utils`?

Reply:
renamed - there was some reason I kept it as test-ext but it doesn't seem to cause any problems being instrumentation